sentences/nb/climate_HassClimateGetTemperature.yaml

```
@@ -2,4 +2,7 @@ language: 'nb'
 intents:
   HassClimateGetTemperature:
     data:
-      - sentences: []
+      - sentences: 
+         - "(hva | hvordan) er temperaturen [i <area>]"
+         - "er det (kaldt | varmt | kjølig) [i <area>]"
+         - "er det (kaldt | varmt | kjølig) inne"
```

This line is the only one we need now. You can use `<kaldt_varmt>`, but it need to be updated to include `kjølig` as well.
```suggestion
         - "er det <kaldt_varmt> inne"
```